indico/worker/uwsgi.ini

```diff
@@ -35,7 +35,6 @@ reload-on-rss = 2048
 evil-reload-on-rss = 8192
 
 offload-threads = 4
-route = ^/robots.txt static:/opt/indico/htdocs/robots.txt
-route = ^/static/custom/(.*) static:/opt/indico/custom/$1
-route = ^/static/assets/(core|(?:plugin|theme)-[^/]+)/(.*)$ static:/opt/indico/assets/$1/$2
-route = ^/(ihelp|css|images|js|static(?!/plugins|/assets|/themes|/custom))(/.*)$ static:/opt/indico/htdocs/$1$2
+# route = ^/robots.txt static:/opt/indico/web/static/robots.txt
```

Comment:
let's remove that?

Reply:
I think it might be used when running on OpenShift. I am not sure but I think that at some point we had problems with nginx serving static files and decided to use `uwsgi` for that.